app/config/collections.php

```diff
@@ -3054,7 +3054,40 @@
                 'required' => false,
                 'default' => null,
                 'filters' => [],
-            ]
+            ],
+            [
+                'array' => false,
+                '$id' => ID::custom('memory'),
+                'type' => Database::VAR_INTEGER,
+                'format' => '',
+                'size' => 512,
+                'signed' => false,
+                'required' => false,
+                'default' => 512,
+                'filters' => [],
+            ],
+            [
+                'array' => false,
+                '$id' => ID::custom('cpus'),
+                'type' => Database::VAR_INTEGER,
+                'format' => '',
+                'size' => 2,
+                'signed' => false,
+                'required' => false,
+                'default' => 1,
+                'filters' => [],
+            ],
+            [
+                'array' => false,
+                '$id' => ID::custom('size'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 128,
+                'signed' => false,
+                'required' => false,
+                'default' => 's-1vcpu-512mb',
+                'filters' => [],
+            ],
```

Comment:
we don't need a size attribute here. `size` is only for the API

Reply:
but if we don't know the machine type, we're going to have to work backwards and recalculate it on the console and work backwards from the CPU's and Memory to show the values allowed

Reply:
Can we only store size, and then get cpus/memory from config, when needed?

Reply:
Yeah, just updated that